data/serializers.py

```diff
@@ -232,7 +232,8 @@ class Meta:
 
 
 class JobTokensSerializer(serializers.ModelSerializer):
+    job = JobSerializer()
     class Meta:
         model = JobToken
         resource_name = 'job-tokens'
-        fields = ('token',)
+        fields = ('id', 'token', 'job')
```

Comment:
Originally I ember js wanted an id but the current bespin-ui  changes don't require it.